crypto/rsa/rsa_gen.c

```diff
@@ -729,3 +729,14 @@ static int rsa_keygen_pairwise_test(RSA *rsa, OSSL_CALLBACK *cb, void *cbarg)
 
     return ret;
 }
+
+#ifdef FIPS_MODULE
+int ossl_rsa_key_pairwise_test(RSA *rsa)
+{
+    OSSL_CALLBACK *stcb;
+    void *stcbarg;
+
+    OSSL_SELF_TEST_get_callback(rsa->libctx, &stcb, &stcbarg);
+    return rsa_keygen_pairwise_test(rsa, stcb, stcbarg);
```

Comment:
I notice that rsa_keygen_pairwise_test() doesnt handle setting a error state. Is this what you intended here?

Reply:
I just reused the existing test without changes.  Do we want to have setting an error state?